### PR TITLE
[ticket/17317] Change button text for pm recipient to increase readability

### PR DESCRIPTION
**Pull request by danieltj27**
I've updated the text on the submit button used to remove recipients from a PM that is currently being composed. It used to just be a small `x` that was hard (for me and of course others) to see.

I've updated it to say `Remove` which is already an existing language string so no new translations are required for this change. I also added a 2px padding so the text isn't flush to the edges of the button.

It makes it more readable and actually looks styled as opposed to looking like an element that was forgotten about. I think this makes the interface much better to use.

PHPBB3-17317

Checklist:

- [x] Correct branch: master for new features; 3.3.x for fixes
- [x] Tests pass
- [x] Code follows coding guidelines: [master](https://area51.phpbb.com/docs/master/coding-guidelines.html) and [3.3.x](https://area51.phpbb.com/docs/dev/3.3.x/development/coding_guidelines.html)
- [x] Commit follows commit message [format](https://area51.phpbb.com/docs/dev/3.3.x/development/git.html)

Tracker ticket:

https://tracker.phpbb.com/browse/PHPBB3-17317

### CODE REVIEW

**Comment by danieltj27** [comment]
I've added a second commit because I accidentally changed `padding: 0;` to `padding: 2px 0;` when it should have been changed to `padding: 0 2px;` instead.

Button need left and right padding, not top and bottom.

**Comment by danieltj27** [comment]
I've updated the change to what @iMattPro has [suggested in the ticket](https://tracker.phpbb.com/browse/PHPBB3-17317?focusedCommentId=65780&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-65780) because I agree that it's a better change compared to simply having a default button with the word remove in it.

File: phpBB/styles/prosilver/template/posting_pm_header.html [hunk 1]
```diff
@@ -48,7 +52,11 @@
 					<ul class="recipients">
 						<!-- BEGIN bcc_recipient -->
 						<li>
-							<!-- IF not S_EDIT_POST --><input type="submit" name="remove_{bcc_recipient.TYPE}[{bcc_recipient.UG_ID}]" value="x" class="button2" /><!-- ENDIF -->
+							<!-- IF not S_EDIT_POST -->
+								<button type="submit" name="remove_{bcc_recipient.TYPE}[{bcc_recipient.UG_ID}]">
+									<i class="icon fa-times icon-red" aria-hidden="true"></i><span class="sr-only">{L_REMOVE}</span>
+								</button>
+							<!-- ENDIF -->
 							<!-- IF bcc_recipient.IS_GROUP --><a href="{bcc_recipient.U_VIEW}" style="color: {{ bcc_recipient.COLOUR }}"><strong>{bcc_recipient.NAME}</strong></a><!-- ELSE -->{bcc_recipient.NAME_FULL}<!-- ENDIF -->
```
Comment (iMattPro) [inline phpBB/styles/prosilver/template/posting_pm_header.html:60]: There's a third one of these down at line 73/81 that needs to be updated.

Comment (danieltj27) [reply phpBB/styles/prosilver/template/posting_pm_header.html:60]: Totally missed that! Done.

**Comment by danieltj27** [comment]
Latest commit addresses both issues in review by @iMattPro.

I'd say it's time to merge and have a brew 🙂

File: phpBB/styles/prosilver/theme/common.css [hunk 2]
```diff
@@ -780,10 +780,10 @@ fieldset.fields1 dd.recipients {
 	margin-left: 1em;
 }
 
-fieldset.fields1 ul.recipients  input.button2 {
+fieldset.fields1 ul.recipients input.button2 {
```
[the remark this answers is not in the document]
Comment (iMattPro) [reply phpBB/styles/prosilver/theme/common.css:783]: None of this rule is needed anymore.

File: phpBB/styles/prosilver/theme/common.css [hunk 3]
```diff
@@ -780,7 +780,7 @@ fieldset.fields1 dd.recipients {
...
```
Comment (iMattPro) [inline phpBB/styles/prosilver/theme/common.css:783]: None of this rule is needed anymore.

Comment (danieltj27) [reply phpBB/styles/prosilver/theme/common.css]: Removed in latest

**Review by iMattPro — APPROVED** [review]

**Comment by marc1706** [comment]
The commit hook check found an issue again. You had for example:
```
[ticket/17317] Some change
PHPBB3-17317
```

However, the commit message hook requires there to be either a comment or an empty line before the PHPBB3-17317:
```
[ticket/17317] Some change

PHPBB3-17317
```

**Comment by danieltj27** [comment]
Thanks, sorry about that. I've done that on a couple of commits but now I know for next time!

**Review by marc1706 — APPROVED** [review]

**Comment by marc1706** [comment]
@danieltj27 On a side note, I have also adapted the changes in master. However, the only difference is that the icon does use a normal cursor for the button now. Might want to check this out and maybe adapt for it accordingly in master.

**Pull request merged**